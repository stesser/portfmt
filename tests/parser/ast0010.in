MAKE_ENV+=     CXXFLAGS_WARN="${CXXFLAGS_WARN}"
MAKE_ENV+=     GNUSED=${LOCALBASE}/bin/gsed
MAKE_ENV+=     DISPLAY=
MAKE_ENV+=     verbose=1
