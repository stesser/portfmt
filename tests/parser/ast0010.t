MAKE_ENV+=     CXXFLAGS_WARN="${CXXFLAGS_WARN}"
MAKE_ENV+=     GNUSED=${LOCALBASE}/bin/gsed
MAKE_ENV+=     DISPLAY=
MAKE_ENV+=     verbose=1
<<<<<<<<<
{ VARIABLE, line 1, .name = MAKE_ENV, .modifier = +=, .words = [1]{ CXXFLAGS_WARN="${CXXFLAGS_WARN}" } }
{ VARIABLE, line 2, .name = MAKE_ENV, .modifier = +=, .words = [1]{ GNUSED=${LOCALBASE}/bin/gsed } }
{ VARIABLE, line 3, .name = MAKE_ENV, .modifier = +=, .words = [1]{ DISPLAY= } }
{ VARIABLE, line 4, .name = MAKE_ENV, .modifier = +=, .words = [1]{ verbose=1 } }
