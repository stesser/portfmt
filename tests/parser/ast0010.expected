{ VARIABLE, line 1, .name = MAKE_ENV, .modifier = +=, .words = { CXXFLAGS_WARN="${CXXFLAGS_WARN}" } }
{ VARIABLE, line 2, .name = MAKE_ENV, .modifier = +=, .words = { GNUSED=${LOCALBASE}/bin/gsed } }
{ VARIABLE, line 3, .name = MAKE_ENV, .modifier = +=, .words = { DISPLAY= } }
{ VARIABLE, line 4, .name = MAKE_ENV, .modifier = +=, .words = { verbose=1 } }
